### PR TITLE
Rf support ubd 55

### PR DESCRIPTION
**Pull request by niranjanraoh**
LPUART driver for RF module is ready for review

### CODE REVIEW

File: firmware/CMakeLists.txt [hunk 1]
```diff
@@ -35,7 +35,7 @@ add_library(firmware
   ubirch/ws2812b.c
   ${CELL_SRCS}
   ${SDHC_SRCS}
-  )
+  ubirch/rf-sub-GHz/rf_lpuart.h ubirch/rf-sub-GHz/rf_lpuart.c)
```
Comment (thinkberg) [inline firmware/CMakeLists.txt]: Please move this up and each file on an individual line, like the other files.

File: firmware/ubirch/rf-sub-GHz/rf_lpuart.c [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+              port_clock_freq);
+  LPUART_EnableRx(rf_config->lpuart, true);
+  LPUART_EnableTx(rf_config->lpuart, true);
+
+  LPUART_EnableInterrupts(rf_config->lpuart, kLPUART_RxDataRegFullInterruptEnable);
+  EnableIRQ(rf_config->lpuart_IRQ);
+  new_rf_config = rf_config;
+}
+
+// Do we need deinit
```
Comment (thinkberg) [inline firmware/ubirch/rf-sub-GHz/rf_lpuart.c]: remove

File: firmware/ubirch/rf-sub-GHz/rf_lpuart.h [hunk 3]
```diff
@@ -0,0 +1,109 @@
+/*!
```
Comment (thinkberg) [inline firmware/ubirch/rf-sub-GHz/rf_lpuart.h]: Check the other files and please add C++ wrapper `extern C {...` together with the defines.

File: tests/board/test_rf_lpuart.c [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#include <ubirch/rf-sub-GHz/rf_lpuart.h>
+
+void test_uart(void) {
+  rf_config_t *rf_config;
+  rf_config = &rf_config_default;
+
+  rf_init(rf_config);
+  const char *hello = "HELLO WORLD\r\n";
+  rf_send(rf_config, (const uint8_t *) hello, 13);
```
Comment (thinkberg) [inline tests/board/test_rf_lpuart.c]: might make sense to use `strlen(hello)` here instead of a magic 13

File: firmware/CMakeLists.txt (hunk 1 above)
Comment (thinkberg) [inline firmware/CMakeLists.txt]: Oh, and make this all small-caps: `rf-sub-ghz`

Comment (niranjanraoh) [inline firmware/CMakeLists.txt]: fixed

File: firmware/ubirch/rf-sub-GHz/rf_lpuart.c (hunk 2 above)
Comment (niranjanraoh) [inline firmware/ubirch/rf-sub-GHz/rf_lpuart.c]: removed

File: tests/board/test_rf_lpuart.c (hunk 4 above)
Comment (niranjanraoh) [inline tests/board/test_rf_lpuart.c]: fixed

**Pull request merged**